api/v1/infinispan_webhook.go

```diff
@@ -147,6 +148,21 @@ func (i *Infinispan) Default() {
 		if i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter == nil {
 			i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter = pointer.Bool(true)
 		}
+		if i.Spec.Service.Sites.Local.Discovery.SuspectEvents == nil {
+			i.Spec.Service.Sites.Local.Discovery.SuspectEvents = pointer.Bool(false)
```

Comment:
If we're always defaulting to false, is there any reason to use a pointer here?

Reply:
I just assumed the pointer was required to use with `omitempty` option: `json:"suspectEvents,omitempty"`